### ISSUE TITLE
Enlargement of eyelid lesion with crying

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10003]
**Preferred term label:**
Enlargement of eyelid lesion with crying

**Synonyms**
Swelling of lesion with crying, includes hemangiomas, varices

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hemangioma, capillary infantile, somatic

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000058
- label: Exacerbation of eyelid lesion by crying
- parents: [Aggravated by (HP:0025285)](https://hpo.jax.org/app/browse/term/HP:0025285)
- pmid: 
- def: Applies to an eyelid lesion such as hemangiomas or varices that swells upon crying.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10003
- synonyms:
